benches/centralized_telescope.rs

```diff
@@ -0,0 +1,201 @@
+//! Benchmarking centralized_telescope
+
+use criterion::{black_box, criterion_group, criterion_main, measurement::WallTime, Criterion};
+use rand_chacha::ChaCha20Rng;
+use rand_core::{RngCore, SeedableRng};
+use std::time::{Duration, Instant};
+
+use alba::centralized_telescope::{algorithm, init, params::Params, setup::Setup};
+
+pub mod criterion_helpers;
+use criterion_helpers::{benchmarks, Repetitions, Steps};
+
+#[path = "../src/utils/test_utils.rs"]
+mod test_utils;
+
+// Global variables
+const NAME: &str = "Centralized";
+// Benchmark parameters
+const L: &[f64] = &[50.0, 128.0]; // Security parameter
+const SP: &[u64] = &[1_000]; // Size of set to lower bound
+const NP: &[u64] = &[80, 90, 95, 98]; // Alba's np parameter, |Sp| >= np
+const NF: &[u64] = &[67, 75]; // Alba's nf parameter,  |Sp| >= np > nf
```

Comment:
Will change np and nf to numbers instead of percentages.

I think there is merit in having the size of the dataset (as defined in the proof) and then send to the prover values between np and |Sp| (that's why I went with percentages instead of the actual numbers).

Reply:
Actually I used _per (as it makes sense to vary the number of element in the prover)